aries-site/src/components/checklist/Checklist.js

```diff
@@ -0,0 +1,79 @@
+import React, { useContext, useState } from 'react';
+import PropTypes from 'prop-types';
+import { FormDown, FormUp, Checkbox, CheckboxSelected } from 'grommet-icons';
+import {
+  AnnounceContext,
+  Anchor,
+  Box,
+  CheckBoxGroup,
+  Paragraph,
+  ResponsiveContext,
+  Text,
+} from 'grommet';
+
+export const Checklist = ({ optionsObjects }) => (
+  <CheckBoxGroup options={optionsObjects}>
+    {(option, { checked }) => {
+      const [expanded, setExpanded] = useState(false);
+      const [isChecked, setChecked] = useState(checked);
+
+      const announce = useContext(AnnounceContext);
+      const size = useContext(ResponsiveContext);
+
+      const Icon = expanded ? FormUp : FormDown;
+      const Check = isChecked ? CheckboxSelected : Checkbox;
+
+      return (
+        <Box direction="row">
+          <Box direction="row" gap="medium" pad="xsmall">
+            <Check
```

Comment:
This doesn't seem right. We should be consuming the Design System's Checkbox instead of reinventing it here.